src/audio/volume/Kconfig.hifi

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+comment "volume optimization level select"
+
+choice "VOLUME_HIFI_LEVEL_SELECT"
+	prompt "choose which hifi level used for volume module"
+
+	config VOLUME_DEFAULT_OPT_LEVEL
```

Comment:
VOLUME_SIMD_NONE

Reply:
I think the intention here is "VOLUME_SIMD_AUTOMATIC" or "VOLUME_SIMD_TOOLCHAIN_MAX". I.e. option that captures the current behaviour, enabled SIMD and use the highest level supported by toolchain.